### PR TITLE
Serwan Asaad nominated to fill seat vacated by Dror Israeli on TSC

### PR DESCRIPTION
**Pull request by jlapeyre**
#### Biographical statement

Serwan Asaad is a product manager at Quantum Machines where he leads the development of software layers above the QUA pulse-level language, which include the calibration software QUAlibrate and the OpenQASM3-to-QUA compiler. He also developed the open-source software QuAM, which can effectively translate gates into hardware-specific pulse-level instructions, and thus facilitates OpenQASM3-to-QUA compilation.

Serwan holds a PhD from UNSW, where he demonstrated the coherent electric control of a high-spin nucleus and achieved record-breaking spin-qubit 2Q gate fidelities. Throughout his academic career, Serwan had a strong focus on developing measurement control software such as [SilQ](https://nulinspiratie.github.io/SilQ/) (not to be confused with Silq), and contributing to the open-source projects QCoDeS and PycQED.

Serwan predicts a transition towards circuit-level languages for quantum control, and OpenQASM3 serves as a necessary bridge in this transition. We're hearing requests from several partners and customers, and we want to promote this integration as much as possible. Ensuring its continued adoption as a standard requires tight collaboration with relevant parties both up and down the stack. Serwan can contribute to this by discussing the challenges and resolutions that arise when integrating OpenQASM3 with hardware-specific languages.

### CODE REVIEW

**Review by braised-babbage — DISMISSED** [review]

File: governance.md [hunk 1]
```diff
@@ -18,10 +18,10 @@ This governance structure is scoped to just OpenQASM itself and not Qiskit as a
 
 ## Structures
 
-We will form a **Technical Steering Committee**. The TSC shall be composed of 6 Contributors. The current TSC (as of January 16, 2024) is:
+We will form a **Technical Steering Committee**. The TSC shall be composed of 6 Contributors. The current TSC (as of June 26, 2024) is:
 
 * Blake Johnson (IBM Quantum)
-* Dor Israeli (Quantum Machines)
+* Serwan Asaad (Quanum Machines)
```
Comment (blakejohnson) [inline governance.md:24]:
```suggestion
* Serwan Asaad (Quantum Machines)
```

**Review by levbishop — DISMISSED** [review]
LGTM welcome Serwan

**Review by levbishop — APPROVED** [review]

**Review by blakejohnson — APPROVED** [review]
I vote in favor of Serwan filling out the balance of Dor's term.

**Review by hodgestar — APPROVED** [review]
I vote in favour of Serwan finishing the remainder of Dor's term.

**Comment by jcqc** [comment]
Agree, I vote in favour of Serwan finishing the remainder of Dor's term.

**Pull request merged**